packages/blockchain-wallet-v4-frontend/src/scenes/Transactions/CustodialTx/model.tsx

```diff
@@ -16,7 +17,16 @@ const Icon = styled(BCIcon)`
   size: 18px;
   font-weight: 600;
 `
+const getSymbolDisplayName = (props: Props) => {
+  return path(
+    [`${props.tx.amount.symbol}`, 'displayName'],
```

Comment:
i dont think you need to do the string template. just the following should work
```
path([props.tx.amount.symbol, 'displayName'],
````